ROMFS/px4fmu_common/init.d/airframes/4053_ifo

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+#
+# @name UVify IFO
+#
+# @output MAIN1 motor 1
+# @output MAIN2 motor 2
+# @output MAIN3 motor 3
+# @output MAIN4 motor 4
+#
+# @board px4fmu-v2 exclude
```

Comment:
Can you add the fmu-v2 exclude to all airframes? Flash is already limited there and it's why the build is currently failing.

Reply:
Done.